contributed_definitions/nyaml/NXxps.yaml

```diff
@@ -0,0 +1,563 @@
+category: application
+doc: |
+  This is the application definition for X-ray photoelectron spectroscopy.
+type: group
+NXxps(NXmpes):
+  (NXentry):
+    definition(NX_CHAR):
+      enumeration: [NXxps]
+    method:
+      doc: |
+        A name of the experimental method according to `Clause 11`_ of
+        the `ISO 18115-1:2023`_ specification.
+        
+        Examples for XPS-related experiments include:
+            * X-ray photoelectron spectroscopy (XPS)
+            * angle-resolved X-ray photoelectron spectroscopy (ARXPS)
+            * ultraviolet photoelectron spectroscopy (UPS)
+            * hard X-ray photoemission spectroscopy (HAXPES)
+            * near ambient pressure X-ray photoelectron spectroscopy (NAPXPS)
+            * electron spectroscopy for chemical analysis (ESCA)
+        
+        .. _ISO 18115-1:2023: https://www.iso.org/standard/74811.html
+        .. _Clause 11: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:sec:11
+    geometries(NXcoordinate_system_set):
+      xps_coordinate_system(NXcoordinate_system):
+        exists: optional
+        doc: |
+          In traditional surface science, the coordinate system is defined such that the positive z-axis
+          points along the sample normal, and the x- and y-axes lie within the sample surface. However,
+          in NeXus, a coordinate system that is the same as `McStas`_  is used. `xps_coordinate_system`
+          gives the user the opportunity to work in the traditional base coordinate system.
+          
+          .. _McStas: http://mcstas.org/
+        origin:
+          enumeration: [sample surface]
+        handedness:
+          enumeration: [right_handed]
+        z_direction:
+          enumeration: [sample normal]
+        depends_on(NX_CHAR):
+          doc: |
+            Link to transformations defining the XPS base coordinate system,
+            which is defined such that the positive z-axis points along the sample normal, and the
+            x- and y-axes lie within the sample surface
+        (NXtransformations):
+          doc: |
+            Set of transformations, describing the orientation of the XPS coordinate system
+            with respect to the beam coordinate system (.).
```

Comment:
The requiredness of this group prevents to add a custom coordinate system which you cannot describe based on "."

Reply:
The idea is that you can then add another coordinate system that is explictly not called `xps_coordinate_system`. I just want to reserve this name and its description for the ISO-defined CS.